headers.js

```diff
@@ -36,11 +36,16 @@ const cspMeta = Object.entries({
     'https://api.coincap.io/v2/assets',
     // @shapeshiftoss/market-service@1.7.0: https://github.com/shapeshift/lib/blob/9123527ebbcf0fd62a619ab2824d970123bd5ac2/packages/market-service/src/coincap/coincap.ts#L21
     'https://api.coincap.io/v2/assets/',
+    // @gem(onramp) to get coinify supported currencies
```

Comment:
plz refer to the gem SDK URL here, the @ convention was related to the npm package names